geode-assembly/src/distributedTest/java/org/apache/geode/management/internal/rest/DeployToMultiGroupDUnitTest.java

```diff
@@ -117,16 +117,17 @@ public void listById() throws Exception {
         assertManagementListResult(list).isSuccessful();
     resultAssert.hasConfigurations().extracting(Deployment::getFileName)
         .containsExactlyInAnyOrder("lib.jar", "lib.jar");
```

Comment:
Doesn't the classpath order matter? This feels like a bad way to compare.

Reply:
I agree with you. The order should matter and `containsExactlyInAnyOrder` seems to disregard this. Given that this file has only cosmetic changes (who knows why the formatting changed now, given that spotless is always applied) and no code changes have been made in this file, I would hope the review would be as such.

I would love to fix all problems in all files that I touch, but in all honesty, I don't have time to fix them all.